share/spack/templates/modules/modulefile.tcl

```diff
@@ -26,9 +26,15 @@ proc ModulesHelp { } {
 {% endblock %}
 
 {% block autoloads %}
+if {![info exists ::env(LMOD_VERSION_MAJOR)]} {
 {% for module in autoload %}
-module load {{ module }}
+    module load {{ module }}
```

Comment:
I think adding indentation reads better in the template, but is weird in the generated module file, right?